packages/editor-ui/src/components/WorkflowActivator.vue

```diff
@@ -120,6 +120,13 @@ export default mixins(
 							return;
 						}
 
+						if (newActiveState) {
+							this.$telemetry.track('User set workflow active status', {
+								user_id: this.$store.getters['users/currentUserId'],
+								instance_id: this.$store.getters.instanceId,
```

Comment:
you don't need to add these. these are included by default

Reply:
Thank you for letting me know. Removed the `user_id` and `instance_id` fields.